python/tvm/autotvm/tophub.py

```diff
@@ -110,6 +110,8 @@ def context(target, extra_files=None):
         device = tgt.attrs.get("device", "")
         if device != "":
             possible_names.append(_alias(device))
+        for k in tgt.keys:
+            possible_names.append(k)
```

Comment:
Please add an explanatory comment to describe what you are trying to fix here.

Also, probably this would work in the same way?
```suggestion
        possible_names.extend(tgt.keys)
```